coderd/database/migrations/fix_migration_numbers.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+list_migrations() {
+	branch=$1
+	git ls-tree -r --name-only "${branch}" -- . | grep -E '[0-9]{6}.*' | sort -n
+}
+
+main() {
+	cd "${SCRIPT_DIR}"
+
+	echo "Fetching origin/main..."
+	git fetch -u origin main
```

Comment:
Is there a project recommendation for origin vs. upstream? I'm developing on my personal fork, so I have to fetch from `upstream`.

```
$ git remote -v
origin	https://github.com/mtojek/coder (fetch)
origin	https://github.com/mtojek/coder (push)
upstream	https://github.com/coder/coder (fetch)
upstream	https://github.com/coder/coder (push)
```

Reply:
Haaaa, I've been hoping nobody has any custom remotes 😂..

I could swap out the hard-coded origin for `origin=$(git remote -v | grep "github.com[:/]coder/coder.*(fetch)" | cut -f1)`, thoughts?

Reply:
Yes, works well!

```
$ git remote -v | grep "github.com[:/]coder/coder.*(fetch)" | cut -f1
upstream
```